app/javascript/packages/verify-flow/steps/personal-key/personal-key-step.tsx

```diff
@@ -0,0 +1,37 @@
+import { PageHeading, Button } from '@18f/identity-components';
+import { useI18n } from '@18f/identity-react-i18n';
+import { FormStepsContinueButton } from '@18f/identity-form-steps';
+
+function PersonalKeyStep() {
+  const { t } = useI18n();
+
+  return (
+    <>
+      <PageHeading>{t('headings.personal_key')}</PageHeading>
+      <p>
+        {t('instructions.personal_key.info_html', {
+          accent: `<strong>t('instructions.personal_key.accent')}</strong>`,
+        })}
```

Comment:
we just got rid of `accent` in https://github.com/18F/identity-idp/pull/6196 right?

Reply:
> we just got rid of `accent` in #6196 right?

Updated in e3c0a68ecdd80f9601d9c9c6538e9ea3503eac8b.